.github/workflows/poc-build-and-runtime-test.yml

```diff
@@ -0,0 +1,102 @@
+# Description: this workflow runs build and simple runtime tests on supported
+#              frameworks and their build systems.
+#
+# Triggered by: it runs every 15 minutes.
+
+name: PoC Build and Runtime Test
+
+on:
+  pull_request:
+    - branches: [main]
```

Comment:
```suggestion
  pull_request:
    branches: [main]
```

Reply:
fyi https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
According to github [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) actually:
```
on:
  pull_request:
    branches:
      - main
  ```

Reply:
Yeah, there're multiple syntax to do it. Either works  😈